drivers/ethernet/Kconfig.liteeth

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (c) 2019 Antmicro
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+menuconfig ETH_LITEETH
+	bool "LiteEth Ethernet core driver"
+	depends on NET_L2_ETHERNET
+
+if ETH_LITEETH
+
+config ETH_LITEETH_0
+	bool "LiteEth Ethernet port 0"
+
+if ETH_LITEETH_0
+
+config ETH_LITEETH_0_IRQ_PRI
+	int "LiteEth interrupt priority"
+	default 0
+	help
+	    IRQ priority
+
+config ETH_LITEETH_0_MAC1
+	hex "MAC address byte 1"
+	default 0x10
```

Comment:
Why do you force this specific mac address to be used? We should have support for "random"/semi-random address as a default, otherwise all the flashed boards will have the same mac address (unless specified separately by the user).

Reply:
It's reworked. The current solution is similar to the one used in other eth drivers - if MAC address randomization is active (that's the default) its lowest 3 bytes are generated using `sys_rand32_get()`.

Reply:
It doesn't solve the problem entirely, though, as the LiteX platform does not provide a true RNG. It is configured with `CONFIG_TEST_RANDOM_GENERATOR`, so the generated MACs will still be the same for all flashed boards.

Reply:
Ok, anyway it is now looking like the other drivers which is good.